environment.yml

```diff
@@ -37,6 +37,7 @@ dependencies:
   - coveralls
   - mypy
   # Useful for jupyter
+  - jupyter
```

Comment:
i am currenly "doubtful" about this, since it can fit a workstyle, or it can't:

I'm not sure if `environment.yml` offers some kind of "optional dependencies" ... but i guess this would be a good candidate for this.

The way i use jupyter is to have one central jupyter installation (in `base` - when using conda) - and add the different environments as interpreters to this central installation, so i only have one "full" jupyter installation (including all plugins and such), and the environments only have `ipykernel` installed.

Note: It's not something i object - but would like to discuss on what you think.

Reply:
> it can fit a workstyle, or it can't:

Yes I agree. I also have a separate environment for my notebook server. Personally, I don't install Jupyter in any projects but I added it to the the installation manifests because I figured less technical users may be confused about that configuration and including it would increase the size of the environment but not impact memory unless the user launches the jupyter server.

I will defer to your decision about including the package.

Reply:
> I'm not sure if `environment.yml` offers some kind of "optional dependencies" ... but i guess this would be a good candidate for this.

As far as I know, Conda does not have concept of optional dependencies. Everything in `environment.yml` will be installed. If your want to support different environment specifications, you need different `environment.yml` files and then control which one is used with options in a setup script or documentation.

Reply:
works for me 👍 , leave it in - as long as we don't use conda to install in docker it won't hurt.